### ISSUE TITLE
Add discussion of "dev" set and "dropout" for regularization

### ISSUE THREAD

**Issue by tyleransom** [tyleransom/DScourseS20#5]
Nowadays they talk about test, dev, training, and validation sets. They also talk about "dropout" as a regularization technique that can reduce both bias *and* variance.
